panacea/src/pml_parser.yrl

```diff
@@ -169,6 +191,45 @@ operator -> '>=' : ">=".
 
 Erlang code.
 
+validate_time({TimeType, [{line, Line}], Val}) ->
+  NewVal = list_to_integer(Val),
+  case TimeType of
+    years when NewVal >= 100 ->
+      return_error(Line, "years cannot be more than 99 (was " ++ integer_to_list(NewVal) ++ ")");
```

Comment:
In all of these you can just refer to Val because you're converting and then converting back?